src/main/java/opendota/Parse.java

```diff
@@ -685,6 +705,19 @@ public void onTickStart(Context ctx, boolean synthetic) {
         return inventoryList;
     }
 
+    private Integer getDotaPlusLevel(int xp) {
```

Comment:
Would it be more flexible to just have the parser output the XP number and then do this mapping at a later stage? Not sure how often it changes

Reply:
Yeah that's probably a good idea.